integration-tests/kudu/pom.xml

```diff
@@ -30,6 +30,13 @@
     <name>Camel Quarkus :: Integration Tests :: Kudu</name>
     <description>Integration tests for Camel Quarkus Kudu extension</description>
 
+    <properties>
+        <!-- This is to allow the "deep reflection" we do in KuduInfrastructureTestHelper -->
+        <opens>java.base/java.net=ALL-UNNAMED</opens>
+        <argLine>--add-opens ${opens}</argLine>
+        <quarkus.native.additional-build-args>-J--add-opens=${opens}</quarkus.native.additional-build-args>
```

Comment:
This is only needed for running the test or have to be used for the user's application?

Reply:
Just the test. The issue is with container based testing.